### ISSUE TITLE
Polybar sometimes don't reads the configuration file.

### ISSUE THREAD

**Issue by stkw0** [jaagr/polybar#1637]
## Describe the issue
Sometimes (seems to be random), polybar starts without reading the configuration, afterwards I have to manually restart polybar.

**Expected behavior:**

Reads configurations

**Actual behavior:**

Only creates a void bar

**Was it working before?**
Not sure.

## To Reproduce
My full config:
```dosini
[bar/bottom]
monitor = HDMI-1

height = 15
padding = 1

override-redirect = true
bottom = true

font-0 = Hack:size=10
font-1 = vlgothic:size=10
font-2 = unifont:size=10

background = #434343

modules-right = date
modules-left = yamp

enable-ipc = true

[bar/top]
monitor = HDMI-1

height = 15

override-redirect = true

font-0 = Hack:size=10
font-1 = vlgothic:size=10

background = #434343

modules-left = ewmh

[module/date]
type = internal/date

interval = 1.0

date = %a %e/%m

time = %k:%M:%S

format = 🕓 <label>
format-background = #434343
format-foreground = #cccccc

label = %date% | %time%
label-foreground = #cccccc

[module/ewmh]
type = internal/xworkspaces

pin-workspaces = false

#label-empty =

format = <label-state>

enable-click = false

enable-scroll = false

[module/yamp]
type = custom/ipc

hook-0 = echo "$(yampc is)($(yampc vg | cut -d. -f 1)%) ==> $(yampc ga) | $(yampc gt)"

initial = 1

[module/yamp-control]
type = custom/text
content = %{A1:yampc p:}⏸%{A}
```
*List any other steps needed to reproduce the issue besides starting polybar with the config you posted above.*

## Polybar Log
Nothing, I increased the log level now, the next time the bug appears I will update this issue with the logs (if any)

## Environment:
* WM: dwm
* Output of `polybar -vvv`:
polybar 3.2.1

Features: +alsa -curl -i3 -mpd -network +pulseaudio +xkeyboard

X extensions: +randr (+monitors) -render -damage -sync -composite +xkb +xrm +xcursor

Build type: Gentoo
Compiler: /usr/bin/x86_64-pc-linux-gnu-g++
Compiler flags: -O2 -pipe -march=native -mtune=native -O2 -flto -Wall -Wextra -Werror -Wno-noexcept-type -O2 -pedantic -pedantic-errors
Linker flags: -Wl,-O1 -Wl,--as-needed -flto

**Comment by kronn** [comment]
Which bar is empty? I can see that you have a top and a bottom bar. If only the top-bar is missing the xworkspaces-module, I would be interested if switching to another workspaces resolves the problem.

**Comment by kronn** [comment]
hmm, in that case, I think I cannot help you and we have to wait for the maintainers or other users to step in. Maybe the log can tell us a better story once you have it :-)

**Comment by patrick96** [comment]
Yeah, I'd need to see a log for that. I doubt that the config file isn't read, it's more likely to be a rendering issue. What distro are you running?

**Comment by stkw0** [comment]
TL;DR: I'm idiot, I will close this issue once I confirm it.

---

The bar was showing so I thought that it wasn't a rendering issue, but after killing polybar I noticed that the bar didn't disappeared. Before using polybar I was using dwm with the dualstatus patch. Interestingly seems that I forget to remove that patch once I switched to polybar, so my problem is probably a data race between dwm and polybar. I will remove that patch and wait some days, just to be sure, afterwards I will close the issue.
